### PR TITLE
Updates the Security map (Ready for Review)

### PR DESCRIPTION
**Pull request by Quantum-M**
Updates the security area to be much nicer and spacious, still a work of progress.

:cl: 
tweak: Brig Map Update.
/:cl:

- [x] Add execution
- [x] Add genpop
- [x] Finish piping and cabling
- [x] Decorate stuff
- [x] Ensure that everything is in order.
---
security nine

Expanded IAA Office
security officers

More formal security offices

securityeight
securityfive
securityfour
securityone
securitysix
Deluxe genpop, for well behaved prisoners
securityten
securitythree
securitytwo

### CODE REVIEW

**Comment by niceping** [comment]
courtroom? genpop = perma?

**Comment by Quantum-M** [comment]
Courtroom is untouched, perma/genpop still being worked on

**Comment by alexkar598** [comment]
Add external cam

**Comment by alexkar598** [comment]
Add external cam for armoury

**Comment by alexkar598** [comment]
Add external cam for armoury

**Comment by alexkar598** [comment]
Add external cam for armoury

**Comment by alexkar598** [comment]
Add external cam for armoury

**Comment by alexkar598** [comment]
Add external cam for armoury

**Comment by alexkar598** [comment]
Add windoor to armoury desk

**Comment by alexkar598** [comment]
More sec lockers would be nice too

**Comment by niceping** [comment]
isnt max sec slots is like 5?

**Comment by ghost** [comment]
Looks nice. I'll slap this onto our next test.

**Comment by Quantum-M** [comment]
Ready for review

**Review by ghost — DISMISSED** [review]
It looks fine to me. Although why does the armory have holy water? But I do like the idea of a "secondary armory". And also a deluxe genpop... I half think that's a good idea, half a "wtf why" idea. But yeah, :+1: from me

File: sunset/code/game/area/Space_Station_13_areas.dm [hunk 1]
```diff
@@ -5,3 +5,35 @@
...
+	ambientsounds = HIGHSEC
+
+/area/security/frontDesk
+	name = "Front Desk"
+	icon_state = "security"
+
+/area/security/toilet
+	name = "Security Toilets"
+	icon_state = "toilet"
+	ambientsounds = HIGHSEC
```
Comment (alexkar598) [inline sunset/code/game/area/Space_Station_13_areas.dm:39]: damm those toilets are safe

**Comment by Quantum-M** [comment]
Removed the holy water flask from armory, added a few holopads around and three modular consoles to security. Also fixed some stuff that was not carry over when conflicts were resolved.

**Review by ghost — APPROVED** [review]
this is fine and high quality

merging

**Pull request merged**